python/ray/tests/test_logging_2.py

```diff
@@ -477,6 +469,27 @@ def print_message(self):
     assert "(MyActor pid=" not in stderr
 
 
+def test_configure_both_structured_logging_and_lib_logging(shutdown_only):
+    """
+    Import `ray.data` to configure the `ray.data` logger. Then, configure the
+    `root` and `ray` loggers in `ray.init()`. Ensure that handlers are not reset.
+    """
+    script = """
+import ray
+import logging
+
+old_test_logger = logging.getLogger("ray.test")
+assert old_test_logger.getEffectiveLevel() != logging.DEBUG
+old_test_logger.setLevel(logging.DEBUG)
+
+ray.init(logging_config=ray.LoggingConfig(encoding="TEXT", log_level="INFO"))
+
+new_test_logger = logging.getLogger("ray.test")
+assert old_test_logger.getEffectiveLevel() == logging.DEBUG
```

Comment:
so with dictConfig, log level will also be cleared not just handlers?

Reply:
Yes, see https://github.com/python/cpython/blob/71ede1142ddad2d31cc966b8fe4a5aff664f4d53/Lib/logging/config.py#L193-L196 for more details. 

Child logger's level will be set to `NOTSET`.